npm/webpack-dev-server/src/helpers/angularHandler.ts

```diff
@@ -120,51 +119,44 @@ export async function generateTsConfig (devServerConfig: AngularWebpackDevServer
 
   const specPattern = Array.isArray(cypressConfig.specPattern) ? cypressConfig.specPattern : [cypressConfig.specPattern]
 
-  const getProjectFilePath = (...fileParts: string[]): string => toPosix(path.join(...fileParts))
-
-  const includePaths = [...specPattern.map((pattern) => getProjectFilePath(projectRoot, pattern))]
+  const includePaths = [...specPattern]
 
   if (cypressConfig.supportFile) {
-    includePaths.push(toPosix(cypressConfig.supportFile))
+    const supportPath = toPosix(path.relative(workspaceRoot, cypressConfig.supportFile))
+
+    includePaths.push(supportPath)
   }
 
   if (buildOptions.polyfills) {
     const polyfills = Array.isArray(buildOptions.polyfills)
       ? buildOptions.polyfills.filter((p: string) => devServerConfig.options?.projectConfig.sourceRoot && p.startsWith(devServerConfig.options?.projectConfig.sourceRoot))
       : [buildOptions.polyfills]
 
-    includePaths.push(...polyfills.map((p: string) => getProjectFilePath(workspaceRoot, p)))
+    includePaths.push(...polyfills.map((p: string) => p))
```

Comment:
```suggestion
    includePaths.push(...polyfills)
```
The map function is not doing anything